src/Table.elm

```diff
@@ -421,6 +423,22 @@ view (Config { toId, toMsg, columns, customizations }) state data =
     sortedData =
       sort state columns data
 
+    caption =
+      case customizations.caption of
+        Nothing ->
+          identity
+
+        Just { attributes, children } ->
+          (::) <| Html.caption attributes children
```

Comment:
Maybe this could be simplified with `Maybe.withDefault`. Like
```
caption = Maybe.map (\{ attributes, children } -> (::) <| Html.caption attributes children)) customizations.caption |> Maybe.withDefault identity
```
Not sure if its actually simpler but it would allow for more reuse if you broke some stuff up into smaller functions.